sources/packages/backend/apps/api/src/services/auth/keycloak/keycloak.service.ts

```diff
@@ -48,7 +53,9 @@ export class KeycloakService {
    */
   public async getOpenIdConfig(): Promise<OpenIdConfig> {
     try {
-      const response = await axios.get(this.authConfig.openIdConfigurationUrl);
+      const response = await firstValueFrom(
```

Comment:
We can directly get a promise using: `httpService.axiosRef.get` instead of getting an Observable and then converting it into a `AxiosResponse` promise. Currently implemented way is fine too.

Reply:
Something like below:

`this.httpService.axiosRef.get(
        this.authConfig.openIdConfigurationUrl,
);`

Reply:
thanks for that, I think, I overlooked it